Formula/gcc.rb

```diff
@@ -89,8 +89,8 @@ def install
 
       make_args = []
       # Use -headerpad_max_install_names in the build,
-      # otherwise lto1 load commands cannot be edited on El Capitan
-      if MacOS.version == :el_capitan
+      # otherwise lto1 load commands cannot be edited
```

Comment:
This line isn't accurate anymore, should probably be something like "otherwise updated load commands won't fit in the Mach-O header."

Reply:
Per @ilovezfs, it would also be good to mention that this is necessary thanks to some evil inside of GCC's build system that escapes our normally mandatory `-headerpad_max_install_names` (see, e.g. https://github.com/Homebrew/brew/blob/master/Library/Homebrew/shims/super/cc#L260-L266)

Reply:
Updated.